test/controllers/api/v2/locations_controller_test.rb

```diff
@@ -397,13 +397,28 @@ def add_locations
     assert_equal location.name, updated_location.name
   end
 
-  test "org admin should not create locations by default" do
-    user = User.create :login => "foo", :mail => "foo@bar.com", :auth_source => auth_sources(:one), :roles => [Role.find_by_name('Organization admin')]
-    as_user user do
-      post :create, params: { :location => { :name => 'loc1'} }
+  test "org admin should create locations by default" do
+    org = taxonomies(:organization1)
+    # Note: org admin role has no default permissions in unit-tests, for real functionality we have to load them before.
+    load File.join(Rails.root, '/db/seeds.d/030-permissions.rb')
+    load File.join(Rails.root, '/db/seeds.d/040-roles.rb')
+    default_org_admin_role = Role.find_by_name('Organization admin')
```

Comment:
There is an org admin role in the fixtures, but it indeed doesn't contain all of the permissions. In this case I think it makes sense to fix the fixture rather then load the files in every test that contains it, since if you're using a role from fixtures you'd expect it to mirror the actual role. (cc: @ares in case there is some reason the role in the fixtures doesn't include the permissions on purpose)

Reply:
@tbrisker @ares I'am not sure if we can (have) to load that roles and permission for unit tests wide usage (think many tests will fail on this) .
Perhaps add a helper function (in test_helper) that will load this roles and permissions.
What do you think?

Reply:
I think that our fixtures should include the state of a newly seeded system, otherwise we might be testing a state that isn't similar to an actual system. I don't think adding more roles or permissions to the fixture should cause any test failures, and if it does that might indicate there is some problem with these tests.